packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -37,11 +41,80 @@ const deskLocaleStrings = {
+  'action.delete.disabled.notReady': 'Operation not ready',
+
+  /** Tooltip when action button is disabled because the document does not exist */
+  'action.delete.disabled.nothingToDelete': 'This document doesn’t yet exist or is already deleted',
+
+  /** Label for the "Delete" document action button */
+  'action.delete.label': 'Delete',
+
+  /** Label for the "Delete" document action while the document is being deleted */
+  'action.delete.running.label': 'Deleting…',
+
+  /** --- DISCARD CHANGES ACTION --- **/
+  /** Tooltip when action button is disabled because the operation is not ready   */
+  'action.discardChanges.disabled.notReady': 'Operation not ready',
+
+  /** Label for the "Discard changes" document action */
+  'action.discardChanges.label': 'Discard changes',
+
+  /** Tooltip when action is disabled because the document has no unpublished changes */
+  'action.discardChanges.disabled.noChange': 'This document has no unpublished changes',
+
+  /** Tooltip when action is disabled because the document is not published */
+  'action.discardChanges.disabled.notPublished': 'This document is not published',
+
+  /** Message prompting the user to confirm discarding changes */
+  'action.discardChanges.confirmDialog.confirmDiscardChanges':
+    'Are you sure you want to discard all changes since last published?',
+
+  /** --- DUPLICATE ACTION --- */
+  /** Tooltip when action is disabled because the operation is not ready   */
+  'action.duplicate.disabled.notReady': 'Operation not ready',
+
+  /** Tooltip when action is disabled because the document doesn't exist */
+  'action.duplicate.disabled.nothingToDuplicate':
+    "This document doesn't yet exist so there‘s nothing to duplicate",
+
+  /** Label for the "Duplicate" document action */
+  'action.duplicate.label': 'Duplicate',
+
+  /** Label for the "Duplicate" document action while the document is being duplicated */
+  'action.duplicate.running.label': 'Duplicating…',
+
+  /** --- UNPUBLISH ACTION --- */
+  /** Tooltip when action is disabled because the operation is not ready   */
+  'action.unpublish.disabled.notReady': 'Operation not ready',
+
+  /** Label for the "Unpublish" document action */
+  'action.unpublish.label': 'Unpublish',
+
+  /** Tooltip when action is disabled because the document is not already published */
+  'action.unpublish.disabled.notPublished': 'This document is not published',
+
+  /** Fallback tooltip for the Unpublish document action when publish is invoked for a document with live edit enabled.*/
+  'action.unpublish.liveEdit.disabled':
+    'This document has live edit enabled and cannot be unpublished',
+
+  /** --- RESTORE ACTION --- */
+  /** Label for the "Restore" document action */
+  'action.restore.label': 'Restore',
+
+  /** Fallback tooltip for when user is looking at the initial version */
+  'action.restore.disabled.cantRestoreInitial': "You can't restore to the initial version",
```

Comment:
I suggest we do not use contractions in identifiers, so I would write this as `action.restore.disabled.cannot-restore-initial`

Reply:
oh, good point - guess we should stick to snake-case as well. Fixed in 8dde36dee0e58072869656ba530a8f8c62286004